front/src/components/login/login.jsx

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { FormErrors } from "./FormErrors.js";
+export class Login extends React.Component {
+	constructor(props) {
```

Comment:
There should be an empty line in between functions, have you configured Prettier?

Reply:
i`m configured Prettier, but I don`t know why this line wasn`t formatted(although I already fixed it)